src/Core/Auth/Services/Implementations/AuthRequestService.cs

```
@@ -276,4 +284,18 @@ private static bool IsDateExpired(DateTime savedDate, TimeSpan allowedLifetime)
     {
         return DateTime.UtcNow > savedDate.Add(allowedLifetime);
     }
+
+    private async Task NotifyAdminsOfDeviceApprovalRequestAsync(OrganizationUser organizationUser, User user)
+    {
+        if (!_featureService.IsEnabled(FeatureFlagKeys.DeviceApprovalRequestAdminNotifications))
+        {
+            return;
```

Do we ever log in cases like this with feature requests or is it worth considering? Feels like silently returning could be confusing if we're ever troubleshooting why an email might not have been sent.

---

We don't usually log this, but that's a great suggestion! Added a warning log.